soh/src/overlays/actors/ovl_Obj_Tsubo/z_obj_tsubo.c

```diff
@@ -226,7 +227,9 @@ void ObjTsubo_SetupWaitForObject(ObjTsubo* this) {
 
 void ObjTsubo_WaitForObject(ObjTsubo* this, PlayState* play) {
     if (Object_IsLoaded(&play->objectCtx, this->objTsuboBankIndex)) {
-        this->actor.draw = ObjTsubo_Draw;
+        if (GameInteractor_Should(VB_POT_DRAW, true, this)) {
```

Comment:
Minor nitpick, but I think this one probably is better named as `VB_POT_SETUP_DRAW` or something equivalent to show that it is not executed _when the pot is drawing_.